client/src/app.jsx

```diff
@@ -1,17 +1,54 @@
-import React from "react";
+import React, { useState } from "react";
 import RelatedProducts from './components/relatedProducts/relatedProductsList.jsx'
 import { Overview } from "./components/overview/GalleryView.jsx"
 import Reviews from "./components/reviews/ReviewsMain.jsx"
 import QandA from './components/questions/QandA.jsx'
 
 
 export const App = () => {
+  var [currentStyleId, setCurrentStyleId] = useState('221014');
+  function handleScroll (e) {
+    let offsetTop  = document.getElementById(e).offsetTop;
+    window.scrollTo({
+      top: offsetTop-100,
+      behavior: "smooth"
+    });
+  }
 
-  return  <div >
-           <h1>Front End Capstone Avatar Project</h1>
-              <Overview />
+  return  (
+          <div>
+            {/* this is the start of the navbar */}
```

Comment:
This makes sense to me, looks good. I'm sure handleScroll is complicated in the background though.